ik_proxy:
  - group_name: 'arm1'
    robot_name: 'arm1'
    tool_name: ''
  - group_name: 'arm1/tool1'
    robot_name: 'arm1'
    tool_name: 'arm1/tool1'
  - group_name: 'arm1/tool2'
    robot_name: 'arm1'
    tool_name: 'arm1/tool2'
  - group_name: 'arm1/camera'
    robot_name: 'arm1'
    tool_name: 'arm1/camera'
  - group_name: 'arm2'
    robot_name: 'arm2'
    tool_name: ''
  - group_name: 'arm2/tool1'
    robot_name: 'arm2'
    tool_name: 'arm2/tool1'
  - group_name: 'arm2/tool2'
    robot_name: 'arm2'
    tool_name: 'arm2/tool2'
  - group_name: 'arm2/camera'
    robot_name: 'arm2'
    tool_name: 'arm2/camera'